10 Sidebar/package.json

```diff
@@ -11,21 +11,21 @@
   "license": "MIT",
   "dependencies": {
     "bootstrap": "~3.3.7",
-    "react": "~15.6.1",
```

Comment:
@ebarriosara , @brauliodiez , do we need all these version changes? Do you agree to update it in class?

Reply:
While I am unsure if this is the reason behind these changes, I can confirm that I have had some issues with the slightly older versions for react and react dom (and the corresponding @types) that prevented me from running the example locally. Thus, I was forced to update to 16.x.x version. It may be that @ebarriosara had similar issues

Reply:
I guess she was experiencing issues as well. Ok then, make sense to me. Go ahead with the conflict fix.

Reply:
Yes, exactly. The previous versions were causing many errors when running, so I have updated them with the same versions that we used in the lessons.